ui/src/components/view/DetailsTab.vue

```diff
@@ -33,6 +33,11 @@
             <router-link :to="{ path: '/volume/' + volume.uuid }">{{ volume.type }} - {{ volume.path }}</router-link> ({{ parseFloat(volume.size / (1024.0 * 1024.0 * 1024.0)).toFixed(1) }} GB)
           </div>
         </div>
+        <div v-else-if="$route.meta.name === 'computeoffering' && item === 'rootdisksize'">
+          <div>
+            {{ parseFloat( resource.rootdisksize / (1024.0 * 1024.0 * 1024.0)).toFixed(1) }} GB
```

Comment:
Suggestion:

I think it would be nice to us to have an utility that centralizes and assures that we always do the same calculation.

@Pearl1594 do you think it would be possible to create an utility to centralize these byte convertions? Like a `getGib(value = 1)` which return `value * 1024 * 1024 * 1024` or `value * GiB` (GiB would be a constant = 1024 * 1024 * 1024).

Reply:
Makes sense.. I can do that. Thanks @GutoVeronezi